0-ad-history-mod-package/simulation/templates/units/han/cavalry_swordsman_b.xml

```diff
@@ -5,6 +5,12 @@
     <SelectionGroupName>units/han/cavalry_swordsman_b</SelectionGroupName>
     <SpecificName>Dāo Qíbīng</SpecificName>
     <Icon>units/han/cavalry_swordsman.png</Icon>
+    <History>
+        For a long time, the sword predominant in Chinese warfare was the Jian, a double-edged sword. However, during the Han dynasty, the Dao (a single-edged sword) slowly replaced the Jian as the military's sword of choice for both cavalry and infantry. This was for the following reasons:
+        Firstly, reliability: the Dao was cheaper and easier to manufacture and maintain, primarily because only one edge was sharpened. It was also more durable and less prone to bending or even breaking. This was again an advantage of the single-edged blade having a wider spine. However, in comparison to other swords, it also had a very broad tang (the back part of the blade inside the hilt), which improved the blade's steadiness additionally.These were crucial factors for the Han military because weapons were mass-produced in numbers reaching hundreds of thousands and handed out to the armies by the state.
+        Secondly, the Dao required—as single-edged swords generally—less training and was easier to master. This was especially important for the Han as well, because their armies largely consisted of militia with little to no combat experience. Another factor was that the early Han were forced to develop a strong cavalry force to counter the cavalry armies of the Xiognu (a nomadic people inhabiting the lands north of the Han Empire). And single-edged swords such as the Dao were more suited for horseback fighting than double-edged swords (such as the Jian).
```

Comment:
```suggestion
        Secondly, the Dao required, as single-edged swords generally do, less training to master. This was important for the Han, because their armies largely consisted of militia with little to no combat experience. Another reason was that the early Han were forced to develop a strong cavalry force to counter the cavalry armies of the Xiognu (a nomadic people inhabiting the lands north of the Han Empire). And single-edged swords such as the Dao were more suited for horseback fighting than double-edged swords (such as the Jian).
```